docs/hardware/devices/rak/peripherals.mdx

```diff
@@ -13,12 +13,32 @@ groupId="rakmodules"
 queryString="rakmodules"
 defaultValue="GPS"
 values={[
+{label: 'User Button', value: 'User Button'},
 {label: 'GPS Module', value: 'GPS'},
 {label: 'Buzzer', value: 'Buzzer'},
 {label: 'I/O Module', value: 'IO'},
 {label: 'Environmental Sensors', value:'Sensors'},
 {label: 'RTC Module', value:'RTC'}
 ]}>
+<TabItem value="User Button">
+
+## User Button
+
+Any tactile momemtary push button is compatible as long as wired to the correct pins, here's a sample of a [Push Button](https://a.co/d/bPhhTxm) 
```

Comment:
missing words "..as long as IT IS wired to.."

Reply:
It's been corrected